pkg/helm/cluster/kube_version.go

```diff
@@ -0,0 +1,30 @@
+package cluster
```

Comment:
do we really need a new package for just single function?

Reply:
Operation of getting kube version is not fitting anywhere in the packages we have in `helm` package, therefore created this package to get cluster related information.

Reply:
I am okay with having this any other package in helm if that fits well.